src/components/CodeEditor/CodeEditor.js

```diff
@@ -65,47 +67,10 @@ class CodeEditor extends Component {
     }
 
     return (
-      <LiveProvider code={showJSX ? code : compiledES6} mountStylesheet={false}>
-        <div
-          css={{
-            [media.greaterThan('xlarge')]: {
-              display: 'flex',
-              flexDirection: 'row',
-            },
-
-            [media.lessThan('large')]: {
-              display: 'block',
-            },
-          }}>
-          {children && (
-            <div
-              css={{
-                flex: '0 0 33%',
-
-                [media.lessThan('xlarge')]: {
-                  marginBottom: 20,
-                },
-
-                '& h3': {
-                  color: colors.dark,
-                  maxWidth: '11em',
-                  paddingTop: 0,
-                },
-
-                '& p': {
-                  marginTop: 15,
-                  marginRight: 40,
-                  lineHeight: 1.7,
-
-                  [media.greaterThan('xlarge')]: {
-                    marginTop: 25,
-                  },
-                },
-              }}>
-              {children}
-            </div>
-          )}
-
+      <div css={{flex: 1}}>
```

Comment:
Gotta wrap it 🙃 I don't think this is used elsewhere, so shouldn't be a big deal!

Reply:
Can't seem to target the `.react-live` class that [`LiveProvider`](https://github.com/FormidableLabs/react-live#liveprovider-) wraps the code in!

That was the solution that I found worked, but targeting from the parent `CodeExample` (in the PR I just made) also works, so that's probably preferable! If `LiveProvider` accepted `className` I'd be able to style directly with the `css` prop 🙃